internal/dag/gatewayapi_processor.go

```diff
@@ -847,6 +855,16 @@ func (p *GatewayAPIProcessor) routes(matchConditions []*matchConditions, headerP
 
 // cluster builds a *dag.Cluster for the supplied set of headerPolicy and service.
 func (p *GatewayAPIProcessor) cluster(headerPolicy *HeadersPolicy, service *Service, weight uint32) *Cluster {
+	if len(p.MinimumTLSVersion) != 0 {
```

Comment:
maybe refactor to something a little less repetetive here (and other places it may apply):

```
cluster := &Cluster{
  Upstream: service
  ...
}

// if minimum tls version and cipher suites set, then set them on `cluster`

return cluster
```

would make it easier if we need to add a new future field, don't have to add it in 2 places

Reply:
Fixed